hera_pspec/uvpspec.py

```diff
@@ -729,28 +830,53 @@ def _clear(self):
         for p in self._all_params:
             if hasattr(self, p):
                 delattr(self, p)
-
+    
+    def __str__(self):
```

Comment:
this is a function that is currently uncovered by travis. we just need a very simple test for this, like not even anything with asserts, just something that runs it and makes sure it finishes with no errors..